keras/src/backend/torch/core.py

```diff
@@ -187,11 +187,7 @@ def __eq__(self, other):
 def convert_to_tensor(x, dtype=None, sparse=None):
     if sparse:
         raise ValueError("`sparse=True` is not supported with torch backend")
-    if type(x) is Variable:
-        # We cannot use `isinstance(x, Variable)` due to the failure of
```

Comment:
It appears to be fixed based on the fact that the unit tests and CI tests pass. See discussion here: https://github.com/keras-team/keras/pull/19955#discussion_r1790558245